### ISSUE TITLE
Infrastructure Upgrade / Mailinglist Info

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#618]
Hi everyone,

the purpose of this ticket is to make everyone aware that the server infrastructure upgrade has been completed and that **you need to take action** by re-registering or re-subscribing as explained in this news item:

https://secretchronicles.org/en/news/2017/10/26/infrastructure/

If you fail to do this, you will not receive information on the mailinglist and/or forum anymore!

/cc @kirbyfan64 @Luiji @xet7 @Bugsbane @DarkAceZ @skarfester @datahead8888 @sydneyjd

Greetings
Marvin

**Comment by kirbyfan64** [comment]
FYI my signup message apparently got stuck in spam... :neutral_face:

**Comment by Quintus** [comment]
@kirbyfan64 xet7 and me have devoted several days into configuring the mailserver with all the bells and whistles one can configure to make it pass spam tests. I sat before this damnit computer from morning till evening just for that stupid topic! I wasted HOURS on it! It consumed my life time! This completely useless big-companies-mock-at-you-small-email-provider topic! AAAARRGH!

If you want to follow an endevour of pure frustration on the topic, check the last day's IRC logs. I am so frustrated on the topic I can't even tell you how much.

**Comment by Quintus** [comment]
I mean, the several last days’ IRC logs. Not yesterday’s. Start here: https://chatlogs.secretchronicles.org/2017-10-22.log.html

**Comment by Quintus** [comment]
Bottom line is: if you aren't big enough, your mail is considered spam. There's nothing one can do against this. Override your spam filter's settings for our lists.

**Comment by Quintus** [comment]
@skarfester @bugsbane -- You two are still not on the new infrastructure. In case you didn't want to read the long text linked, the tl;dr for webinterface users is that you need to go to https://lists.secretchronicles.org/accounts/signup/?next=/postorius/lists/ , create an account on that page and then use the website at https://lists.secretchronicles.org to interact with the lists. We as the dev team would like to not lose you two, because the game is void without graphics.

**Comment by Bugsbane** [comment]
I've created an account there although I'm not subscribed to either list. In the past I've found that mailing lists flood my inbox very quickly.

This is why I've always preferred forums and GitHub, as they let me take an opt in approach to getting messages (ie I get nothing unless I request it for something relevant), rather than an opt out approach (ie I get sent everything unless I delete it).

I'll take a peek at the archives from time to time, but if you really need to alert me to something, GitHub or a direct email are still the best ways.
